backend/yki/src/main/java/fi/oph/yki/api/clerk/ClerkCustomerDetailsController.java

```diff
@@ -0,0 +1,30 @@
+package fi.oph.yki.api.clerk;
+
+import static org.springframework.http.MediaType.ALL_VALUE;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+import fi.oph.yki.api.dto.clerk.*;
+import fi.oph.yki.service.ClerkCustomerService;
+import io.swagger.v3.oas.annotations.Operation;
+import jakarta.annotation.Resource;
+import java.util.List;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(value = "/v2/api/clerk/customer", consumes = APPLICATION_JSON_VALUE, produces = APPLICATION_JSON_VALUE)
+public class ClerkCustomerDetailsController {
```

Comment:
```suggestion
@Profile("dev")
public class ClerkCustomerDetailsController {
```

Pitänee keksiä myöhemmin jokin keino, miten saadaan käyttöön vain testiympäristöissä, mut toistaiseksi voisi laittaa tämän rajauksen, ettei tätä pysty käyttämään tuotannossa kun ei ole vielä auditoitu.